pkg/runtime/interfaces/application_configuration.go

```diff
@@ -59,6 +61,19 @@ type ApplicationConfig struct {
 	// This is useful to achieve fairness. Note: MapTasks are regarded as one unit,
 	// and parallelism/concurrency of MapTasks is independent from this.
 	MaxParallelism int32 `json:"maxParallelism"`
+	// Labels to apply to the execution resource.
+	Labels map[string]string `json:"labels,omitempty"`
+	// Annotations to apply to the execution resource.
+	Annotations map[string]string `json:"annotations,omitempty"`
+
+	// Optional: security context override to apply this execution.
+	// iam_role references the fully qualified name of Identity & Access Management role to impersonate.
+	AssumableIamRole string `json:"assumableIamRole"`
```

Comment:
nit: add comments to pflag?

Reply:
We are not using this as pflag config and none of the other fields in this config can be passed from command line.
If we want that behavior then i would need to add a pflag generator and add pflag annotation to all.
Let me know if we need that now or in any future PR's

Reply:
ah okay, nevermind then